### ISSUE TITLE
New data-driven fish crash

### ISSUE THREAD

**Issue by sleepingdragoninn** [Pyxrs/Live-Fishing#10]
Happens on loading world or clicking create world. Can isolate further if not reproduceable

```
Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_7923 	at 
com.pyxrs.fishing.data.serialization.BasicFish.getItemStack(BasicFish.java:36) ~[fishing-1.2.0.jar:?] 	at 
com.pyxrs.fishing.data.serialization.BasicFish$Serializer.deserialize(BasicFish.java:77) ~[fishing-1.2.0.jar:?] 	at 
com.pyxrs.fishing.data.serialization.NamedFish$Builder.deserialize(NamedFish.java:53) ~[fishing-1.2.0.jar:?] 	at 
com.pyxrs.fishing.data.FishManager.lambda$reload$1(FishManager.java:79) ~[fishing-1.2.0.jar:?]
```

**Comment by Pyxrs** [comment]
very strange... This was code made by a contributor that I do not fully understand, so I may just remove the data-driven part of the mod

**Comment by Pyxrs** [comment]
Fixed in 1.2.1
